pyracing/response_objects/historical_data.py

```diff
@@ -181,7 +181,7 @@ def __init__(self, dict):
 class PrivateResults:
     def __init__(self, dict):
         self.car_class_id = dict['carclassid']
-        self.car_ids = dict['carids']
+        self.car_ids = parse_encode(dict['carids'])
```

Comment:
This specific result comes as a string of comma separated numbers that are encoded as URL %2C.